### DISCUSSION TITLE
Please seed TG799vac / VANT-R, 16.2.7732

### DISCUSSION THREAD

**Discussion by kyberias** [hack-technicolor/hack-technicolor#344] [Q&A]
Despite what the Firmware Repository states, versions 17.2 don't seem to be rootable with strategy #A, so I would like to get my hands on the previous 16.2. version:

1627732r2221004closed.rbi

Please seed if you have the file available. Thanks.

**Comment by subjoi** [comment]
Bumping. Please Seed any of the rootable VANT-R firmwares. Thank you

**Reply by subjoi** [reply]
When I first bumped this topic I was unprepared to downgrade my Type 3 firmware on the router. Since then I've got a dc/dc power modulator to help me bootfail the router and change active boot bank.
I've lost the VANT-R firmwares unfortunately and would very much appreciate any seeds.

I'll be backing them up and seeding them for the foreseeable future ahead. Thanks

**Reply by kyberias** [reply]
Seeding.
